content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -17,24 +17,15 @@ weight: 40
 
 For clarity, this guide defines the following terms:
 
-Node
-: A worker machine in Kubernetes, part of a cluster.
-
-Cluster
-: A set of Nodes that run containerized applications managed by Kubernetes. For this example, and in most common Kubernetes deployments, nodes in the cluster are not part of the public internet.
-
-Edge router
-: A router that enforces the firewall policy for your cluster. This could be a gateway managed by a cloud provider or a physical piece of hardware.
-
-Cluster network
-: A set of links, logical or physical, that facilitate communication within a cluster according to the Kubernetes [networking model](/docs/concepts/cluster-administration/networking/).
-
-Service
-: A Kubernetes {{< glossary_tooltip term_id="service" >}} that identifies a set of Pods using {{< glossary_tooltip text="label" term_id="label" >}} selectors. Unless mentioned otherwise, Services are assumed to have virtual IPs only routable within the cluster network.
+* Node: A worker machine in Kubernetes, part of a cluster.
+* Cluster: A set of Nodes that run containerized applications managed by Kubernetes. For this example, and in most common Kubernetes deployments, nodes in the cluster are not part of the public internet.
+* Edge router: A router that enforces the firewall policy for your cluster. This could be a gateway managed by a cloud provider or a physical piece of hardware.
+* Cluster network: A set of links, logical or physical, that facilitate communication within a cluster according to the Kubernetes [networking model](/docs/concepts/cluster-administration/networking/).
+* Service: A Kubernetes {{< glossary_tooltip term_id="service" >}} that identifies a set of Pods using {{< glossary_tooltip text="label" term_id="label" >}} selectors. Unless mentioned otherwise, Services are assumed to have virtual IPs only routable within the cluster network.
 
 ## What is Ingress?
 
-Ingress exposes HTTP and HTTPS routes from outside the cluster to
+[Ingress](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#ingress-v1beta1-networking-k8s-io) exposes HTTP and HTTPS routes from outside the cluster to
```

Comment:
Line 28, I wonder whether this link is needed here? Provide the link to the resource API in the `What's next` section.
Line 468, suggested edit: [Learn about the Ingress resource](...) OR [Learn about the Ingress API](...)

Reply:
For line 28, I think it's nice to have, our users see `What is Ingress` section and could easily visit the detail api fields via this link.

For line 468, I'm ok to do this change.

Reply:
Hello @Colstuwjx, A couple of thoughts:
Personally, I am not a big fan of starting a paragraph with linked text (and the link to the API is listed with the other reference links at the bottom of the page).
I think your updates to the formatting of the terminology section is OK (granted the current docs UI
does not provide custom styling for this type of component).
Since the docs are in the process of changing to a new theme/UI, this page section will hopefully
get an improved look when that change lands.
/lgtm

Reply:
Okay, I got the point, thanks for your suggestions!